test/number/format-decimal.js

```diff
@@ -152,6 +152,19 @@ describe('NumberFormat', function() {
 				expect(value).to.equal('10,00000');
 			});
 
+			it('should handle group size of 0', function() {
+				localeData.groupSize = 0;
+				var options = {maximumFractionDigits: 2};
+				var value = format(1000000.01, localeData, options);
+				expect(value).to.equal('1000000.01');
+			});
+
+			it('should handle variable group sizes', function() {
+				localeData.groupSize = [ 3, 2, 0 ];
+				var options = {maximumFractionDigits: 2};
+				var value = format(1000000.01, localeData, options);
+				expect(value).to.equal('10,00,000.01');
+			});
```

Comment:
Based on [Microsoft's documentation](https://msdn.microsoft.com/en-us/library/system.globalization.numberformatinfo.numbergroupsizes(v=vs.110).aspx), it should use the last group size value indefinitely. So maybe a test case for that? e.g. groupSize = [4,2], input = "123456789.01", output = "1,23,45,6789.01"